### PR TITLE
[TEST] fix some issues with simd tests on other platforms

### PR DESCRIPTION
**Pull request by marehr**
fix some issues with simd tests on other platforms (see #1381) and I also removed the namespace seqan3.

### CODE REVIEW

File: test/unit/core/simd/simd_test.cpp [hunk 1]
```diff
@@ -9,44 +9,44 @@
...
 #include <seqan3/core/simd/all.hpp>
 
-using namespace seqan3;
-
-using int16x_t = seqan3::simd::simd_type<int16_t>::type;
-
 TEST(simd, auto_length)
 {
-    if constexpr(simd_traits<int16x_t>::max_length == 64)
+    using int16x_t = seqan3::simd::simd_type<int16_t>::type;
```
Comment (rrahn) [inline test/unit/core/simd/simd_test.cpp]: can you not use using seqan3::simd within the test body? It spoils the entire test. Or is this a general problem?

Comment (marehr) [reply test/unit/core/simd/simd_test.cpp]: you mean `using namespace seqan3::simd`? sure I can xD

Comment (marehr) [reply test/unit/core/simd/simd_test.cpp]: We fully specify everything in tests now :)

Comment (rrahn) [reply test/unit/core/simd/simd_test.cpp]: 😭

**Pull request merged**